hardhat.config.ts

```diff
@@ -1,11 +1,13 @@
 import '@nomicfoundation/hardhat-chai-matchers';
+import '@protocolink/test-helpers';
```

Comment:
```suggestion
```

Reply:
The import is used to load custom chai matchers. You can refer to the following link for more information:
https://github.com/dinngo/protocolink-js-sdk/blob/master/packages/test-helpers/src/index.ts#L1

Reply:
removed~